### ISSUE TITLE
Question about downstream task of object detection?

### ISSUE THREAD

**Issue by ashutoshsingh0223** [PRBonn/segcontrast#21]
When reporting the results of validation split of KITTI dataset, do you use the best model or final model after supervised finetuning?

**Comment by ashutoshsingh0223** [comment]
I am actually trying to reproduce results across various methods similar to yours and I notice that the results of training from scratch vary significantly.

**Comment by nuneslu** [comment]
Hi, I have used the same strategy used in the object detection code from this [repo](https://github.com/zaiweizhang/OpenPCDet). You are right, this is also something I noticed, my guess is that the object detection methods are usually based on non-learned post-processing methods which are based on thresholds, which then can be quite sensible to minimal changes in the features learned by the network.

**Comment by ashutoshsingh0223** [comment]
I realized this from the paper that you used the same settings from DepthContrast, however my question is about the reported metrics. On the validation set do you select the best model or model from the final epoch?

I mean are these the best metrics by evaluating during training or from the last epoch?

**Comment by nuneslu** [comment]
From all the methods we've got the best model

**Comment by ashutoshsingh0223** [comment]
Okay thankyou, just for confirmation, you trained(finetuned) PointRCNN and reported the best mAP? Basically there was model selection involved, is this correct?

I am sorry for bothering but I just want to know the standard the practices, because your paper is one of most influential ones, especially when pretraining for detection/segmentation tasks.

**Comment by nuneslu** [comment]
Hi again! Sorry, I've got confused. Actually, there was no model selection involved, we just got the last checkpoint from all the methods.

**Comment by ashutoshsingh0223** [comment]
Okay. That's what I wanted to know. Thankyou!
